Explicitly identify missing ingress spans could be due to experimenta… (#5266)
istio/istio.io: content/en/faq/distributed-tracing/no-tracing.md

If you still do not see any trace data, please confirm that your ports conform to the Istio [port naming conventions](/faq/traffic-management/#naming-port-convention) and that the appropriate container port is exposed (via pod spec, for example) to enable
traffic capture by the sidecar proxy (Envoy).

> If you only see trace data associated with the egress proxy, but not the ingress proxy, it may still be related to the Istio [port naming conventions](/faq/traffic-management/#naming-port-convention). However, from [Istio 1.3](/news/2019/announcing-1.3/#intelligent-protocol-detection-experimental) the protocol for outbound traffic is automatically detected.

Reviewer: nit: to simplify, I'd remove `However` and emphasize **outbound**.

Do we want to add something about expectations for protocol detection?

Author: Updated.

> If you only see trace data associated with the egress proxy, but not the ingress proxy, it may still be related to the Istio [port naming conventions](/faq/traffic-management/#naming-port-convention). From [Istio 1.3](/news/2019/announcing-1.3/#intelligent-protocol-detection-experimental) the protocol for **outbound** traffic is automatically detected. In the future protocol detection should also be supported for inbound traffic.

Reviewer 2: In general, we try to avoid statements about the future in the docs. So I'd change this to:

```suggestion
If you only see trace data associated with the egress proxy, but not the ingress proxy, it may still be related to the Istio [port naming conventions](/faq/traffic-management/#naming-port-convention). Starting with [Istio 1.3](/news/2019/announcing-1.3/#intelligent-protocol-detection-experimental) the protocol for **outbound** traffic is automatically detected.
```
